searx/settings.yml

```diff
@@ -67,11 +67,13 @@ ui:
 #    key : !!binary "your_morty_proxy_key"
 
 outgoing: # communication with search engines
-    request_timeout : 2.0 # default timeout in seconds, can be override by engine
+    request_timeout : 3.0 # default timeout in seconds, can be override by engine
     # max_request_timeout: 10.0 # the maximum timeout in seconds
     useragent_suffix : "" # suffix of searx_useragent, could contain informations like an email address to the administrator
-    pool_connections : 100 # Number of different hosts
-    pool_maxsize : 10 # Number of simultaneous requests by host
+    pool_connections : 100 # The maximum number of concurrent connections that may be established.
+    pool_maxsize : 20 # Allow the connection pool to maintain keep-alive connections below this point.
+    keepalive_expiry: 30.0 # Number of seconds to keep a connection in the pool
+    http2: True  # Enable HTTP/2 (experimental)
```

Comment:
If it is experimental why enable it? What is the impact of enabling this?

Reply:
See https://www.python-httpx.org/http2/ 

Using `AsyncHTTPTransportFixed`, I haven't seen issue on my local instance for 2 weeks.